src/mixins/enum-mixin.ts

```diff
@@ -1,30 +1,12 @@
 import Vue from 'vue'
 import { Component } from 'vue-property-decorator'
-import {
-  DissolutionTypes,
-  EffectOfOrderTypes,
-  FilingStatus,
-  FilingTypes,
-  PaymentMethod
-} from '@/enums'
-import {
-  GetCorpFullDescription,
-  GetCorpInfoObject,
-  GetCorpNumberedDescription
-} from '@bcrs-shared-components/corp-type-module'
-import EnumUtilities from '@/services/enum-utilities'
-import { Getter } from 'vuex-class'
+import { EffectOfOrderTypes, FilingStatus, FilingSubTypes, FilingTypes, PaymentMethod } from '@/enums'
 
 /**
  * Mixin that provides some useful enum-related utilities.
  */
 @Component({})
 export default class EnumMixin extends Vue {
```

Comment:
Is it possible to delete this file since it's deprecated now?

Reply:
There are still many code references to this mixin. I've only moved some of them over (as a favour to my reviewers so they don't have to review a gazillion files).